Add missing feature screenshots

hokein opened the pull request.

hokein committed:
Move png files to a dedicated directory.

hokein committed:
Add missing screenshots for the features.

mirror from the llvm clangd page.

sam-mccall commented:
Thanks for collecting these! A few bits throughout:

`doc-asserts/` => `screenshots/` or so? Assets is more general and I don't think we should move CSS or even other images there, necessarily.

Can we match the existing naming convention (snake_case apply_fix.png) and drop the InVSCode suffix for each feature? Screenshots from VSCode except where explicitly demoing another editor seems like a good policy.

I don't think the screenshots should be behind zippies on the features page. Everyone loves screenshots!

I'd be nice to have non-empty alt text for the images (e.g. "screenshot: apply fix" or something).

(FWIW the new images don't show up in the code review preview of the MD file and the old ones do, maybe related to missing alt text?)

hokein committed:
address review comments:
 - doc-asserts => screenshots
 - canonicalize screenshot names
 - removed the span

hokein commented:
> doc-asserts/ => screenshots/ or so? Assets is more general and I don't think we should move CSS or even other images there, necessarily.

Done.

> Can we match the existing naming convention (snake_case apply_fix.png) and drop the InVSCode suffix for each feature? Screenshots from VSCode except where explicitly demoing another editor seems like a good policy.

Done.

> I don't think the screenshots should be behind zippies on the features page. Everyone loves screenshots!

Done.

> I'd be nice to have non-empty alt text for the images (e.g. "screenshot: apply fix" or something).

Done.

> (FWIW the new images don't show up in the code review preview of the MD file and the old ones do, maybe related to missing alt text?)

oh, it worked locally (`jekyll serve`), looks like github preview requires the alt text to show the images.
